src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1101,6 +1249,13 @@ namespace
                     color_name,
                     intensity);
             }
+            else if (object_state.obj->IsSubClassOf(LIGHTSCAPE_LIGHT_CLASS))
```

Comment:
I can't comment there, but could you fix indentation on line 1243 please?

Reply:
Done.